packages/atomic/src/components/common/no-items/container.ts

```diff
@@ -0,0 +1,10 @@
+import {FunctionalComponentWithChildren} from '@/src/utils/functional-component-utils';
+import {html} from 'lit';
+
+export const renderNoItemsContainer: FunctionalComponentWithChildren<{}> =
+  () => (children) => html`
+    <div class="text-on-background flex h-full w-full flex-col items-center">
+      ${children}
+    </div>
+    <slot></slot>
```

Comment:
We're missing tests for the slot, which is what ultimately allows users to inject additional content in the `atomic-no-products` component.

I think we should at least test that the component renders the slot outside of the div.